rest_api/rest_api/controller/health.py

```diff
@@ -0,0 +1,126 @@
+from typing import List
+
+import logging
+
+import os
+import pynvml
+import psutil
+
+from pydantic import BaseModel, Field
+
+from fastapi import FastAPI, APIRouter
+
+import haystack
+
+from rest_api.utils import get_app
+from rest_api.config import LOG_LEVEL
+
+logging.getLogger("haystack").setLevel(LOG_LEVEL)
+logger = logging.getLogger("haystack")
+
+
+router = APIRouter()
+app: FastAPI = get_app()
+
+
+class CPUUsage(BaseModel):
+    avg_1m: str = Field(..., description="Average CPU usage over the last 1 minute")
+    avg_5m: str = Field(..., description="Average CPU usage over the last 5 minutes")
+    avg_15m: str = Field(..., description="Average CPU usage over the last 15 minutes")
```

Comment:
On a second thought, I think using `str` here simply to be able to add the `%` sign makes it unnecessary hard to hook this endpoint to a monitoring tool. I understand this health endpoint is more for human consumption than for machines, but sticking to fractions of 1 instead of using formatted percentages has the nice side effect of making it easier to dump the data in some nice plot if there's the need.

What do you think? Are you ok changing these (and most of the other fields below) back to `float`?

Reply:
Float is easier for machine readability. The consumer won't need to parse it. 

And the original thoughts weren't really for human. You are right. Let's change it.